README.md

```diff
@@ -67,7 +62,7 @@ If you want to build Kubernetes right away there are two options:
 ```
 $ go get -d k8s.io/kubernetes
 $ cd $GOPATH/src/k8s.io/kubernetes
-$ make
```

Comment:
probably should leave this one as-is. it's intentionally not using the docker build.